### PR TITLE
Fix prediction results table logic error

### PR DESCRIPTION
**Pull request by acrellin**
Still to do:
- Make sure objects/arrays are not null before attempting to map/index etc.

### CODE REVIEW

File: public/scripts/Predictions.js [hunk 1]
```diff
@@ -162,7 +163,7 @@ let PredictionResults = (props) => {
 
             <td>{fname}</td>
 
-            {hasTarget(result) && [
+            {[hasTrueTargetLabel(result) &&
```
Comment (stefanv) [inline public/scripts/Predictions.js:166]: Logic error was here, I presume, for those watching along.

Comment (stefanv) [inline public/scripts/Predictions.js:166]: This doesn't make sense to me any more, though :)

Comment (stefanv) [inline public/scripts/Predictions.js:166]: OK, looks fine to me.

Comment (acrellin) [inline public/scripts/Predictions.js:166]: Everything else would be omitted if it evaluates to false!...

**Pull request merged**